pkg/public/util.go

```diff
@@ -1,12 +1,14 @@
-package resources
+package public
```

Comment:
`client` could be an alternative name to `public`, unless there's an issue with that